app/views/notification_mailer/application_created.html.slim

```diff
@@ -14,4 +14,8 @@ ul
       b #{@application.field_label(field.name)}:
       |  #{@application.field_value(field.name).presence || 'not provided by user'}
 
+- if @plan.approval_required?
+  p
+    | This service requires you to approve this application before it can be used.
```

Comment:
Shouldn't we use the translation file here?

Reply:
I don't think so because this is a template for an email notification, the rest for the template is not using the translation file, nor any other email template on the codebase AFAIK.